content/news/call-for-sponsors.md

```diff
@@ -0,0 +1,27 @@
++++
+title = 'Call for Sponsors'
+date = 2024-08-16T20:26:21-05:00
+draft = true
```

Comment:
```suggestion
draft = false
```

Reply:
It doesn't seem to show up on the preview when the draft is set to True

Reply:
I guess that makes sense... I didn't know if it was to be set to true or false from the start or not. Now I guess I do. :-)